projects/step-core/src/lib/modules/table/shared/table-remote-data-source.ts

```diff
@@ -36,6 +37,7 @@ export class TableRequest {
     this.start = data?.start || undefined;
     this.length = data?.length || undefined;
     this.filter = data?.filter || undefined;
+    this.params = data?.params || undefined;
```

Comment:
I know it is not new, but isnt the undefined redundant? If either data or params is not defined it will be undefined.

Reply:
It's just a cleanup in case if parameter will be send as null, or empty string.
For example without it, we could receive the following case in theory, that request will contain:
```
{
  "filter": "",
  "params": null
}
```
And it may affect on the response's result.

With undefined replace, such fields won't be included in the requests json body.